web/html/src/components/formulas/EditGroup.tsx

```diff
@@ -41,10 +47,16 @@ class EditGroup extends React.Component<EditGroupProps, EditGroupState> {
   constructor(props: EditGroupProps) {
     super(props);
     this.state = {
-      visible: true,
+      visible: props.sectionsExpanded !== "collapsed",
     };
   }
 
+  componentDidUpdate(prevProps: Readonly<EditGroupProps>) {
```

Comment:
Just as a heads up, since `setSectionsExpanded` is an inline function, this might fire more often than you perhaps planned, but I might be worrying too much, I don't know the business logic here and this might be completely fine.